runtime/altair/src/lib.rs

```diff
@@ -581,6 +582,12 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 				RuntimeCall::Permissions(pallet_permissions::Call::add { .. })
 					| RuntimeCall::Permissions(pallet_permissions::Call::remove { .. })
 			),
+			ProxyType::Transfer => {
+				matches!(
+					c,
+					RuntimeCall::XTokens(..) | RuntimeCall::Balances(..) | RuntimeCall::Tokens(..)
```

Comment:
@cdamian sit, I forgot something. We should add two more here

`RuntimeCall::LiquidityPools(pallet_liquidituy_pool::Call::Transfer{..} | pallet_liquidituy_pool::Call::TransferTrancheTokens{..})`